start.sh

```diff
@@ -8,9 +8,33 @@ SIZE='Standard_D8s_v4'
 GROUPNAME='mob'
 USERNAME='mob'
 PASSWORD='mob'
-MYIP=$(curl ipinfo.io/ip)"/32" # if not given as a parameter it will take your outgoing IP/32
+MYIP=$(curl -sq ipinfo.io/ip)"/32" # if not given as a parameter it will take your outgoing IP/32
```

Comment:
Manpage says about "-q": "If used as the first parameter on the command line, the curlrc config file will not be read and used." So I think you need to put -q as the first parameter so it will have any effect!?

Reply:
typo, needs only "-s" but anyway doesn't have any effect in my environment as I don't use curlrc.